src/js/lib/monster.routing.js

```diff
@@ -104,12 +104,20 @@ define(function(require) {
 		},
 
 		addDefaultRoutes: function() {
-			this.add('apps/{appName}:?query:', function(appName, query) {
+			this.add(/^apps\/([^/?]+)(\/[^?]*)?\??(.*)?$/, function(appName, restSegment, queryString) {
```

Comment:
Since app name characters are in the range [a-z-], what do you thing about restricting the first capturing group to that specific pattern?

`^apps\/([a-z]+(?:-[a-z]+)?)(\/[^?]*)?\??(.*)?$`

![image](https://user-images.githubusercontent.com/1958193/174417216-83c299dd-8eff-4df4-b235-9b3b2a5cf764.png)

Reply:
Yes, sounds good to me, that will ensure that the app name has the right pattern 👍